src/main/java/com/dougnoel/sentinel/pages/PageData.java

```diff
@@ -91,8 +92,14 @@ public static PageData loadYaml(File fileName) throws IOException{
      * @return Map&lt;String, String&gt; the locators for an element
      */
     public Map<String,String> getElement(String elementName) {
-    	if (elements.containsKey(elementName)) {
-    		return elements.get(elementName);
+    	String pageName = PageManager.getPage().getName();
```

Comment:
Move this  into line 101 and don't declare a variable, just put the call to PageManager.getPage().getName() inline.

Reply:
done